epub33/core/index.html

```diff
@@ -379,6 +379,12 @@ <h3>Terminology</h3>
 							conformant with this specification.</p>
 					</dd>
 
+					<dt><dfn id="dfn-epub-validator" data-lt="EPUB Validators">EPUB Validator</dfn></dt>
```

Comment:
I personally prefer "conformance checker" instead of "validator". It is closer to spec language and is a bit less grim.